Compile Issue

```
Saul Nachman12:17 PM
# command-line-arguments
./test.go:31:2: imported and not used: "k8s.io/api/apps/v1" as appsv1
./test.go:32:2: imported and not used: "k8s.io/api/core/v1" as corev1
./test.go:33:2: imported and not used: "k8s.io/api/networking/v1" as networkingv1
./test.go:35:2: imported and not used: "k8s.io/apimachinery/pkg/util/intstr"
./test.go:73:25: undefined: Ingress
./test.go:108:33: client.NetworkingV1().Ingresss undefined (type "k8s.io/client-go/kubernetes/typed/networking/v1".NetworkingV1Interface has
```

--- a/codify.go
+++ b/codify.go
@@ -370,8 +370,10 @@ func toCodify(raw []byte) ([]CodifyObject, error) {
 		objects = append(objects, codify.NewPodSecurityPolicy(x))
 	case *admissionregistrationv1.ValidatingWebhookConfiguration:
 		objects = append(objects, codify.NewValidatingwebhookConfiguration(x))
+	case *policyv1.PodDisruptionBudget:
+		objects = append(objects, codify.NewPodDisruptionBudget(x))
 	// CRDs is going to take some special care...
-	//case *apiextensionsv1.CustomResourceDefinition:
+	case *apiextensionsv1.CustomResourceDefinition:
 	// ignore CRDs for now!
 	//objects = append(objects, codify.NewCustomResourceDefinition(x))
 	case *corev1.Namespace:

--- a/codify/codify.go
+++ b/codify/codify.go
@@ -70,6 +70,7 @@ var (
 		"SecretProjection",
 		"ConfigMapKeySelector",
 		"ConfigMapProjection",
+		"ConfigMapEnvSource",
 		"HTTPHeader",
 		"PodAntiAffinity",
 		"PodAffinityTerm",

--- a/codify/codify.go
+++ b/codify/codify.go
@@ -67,6 +67,7 @@ var (
 
 	CoreV1Types = []string{
 		"Volume",
+		"SecretKeySelector",
 		"SecretProjection",
 		"ConfigMapKeySelector",
 		"ConfigMapProjection",

--- a/codify/ingress.go
+++ b/codify/ingress.go
@@ -74,12 +74,13 @@ func (k Ingress) Install() (string, []string) {
 func (k Ingress) Uninstall() string {
 	uninstall := `
 	if client != nil {
-		err = client.NetworkingV1().Ingress("{{ .KubeObject.Namespace }}").Delete(context.TODO(), "{{ .KubeObject.Name }}", metav1.DeleteOptions{})
+		err = client.NetworkingV1().Ingresses("{{ .KubeObject.Namespace }}").Delete(context.TODO(), "{{ .KubeObject.Name }}", metav1.DeleteOptions{})
 		if err != nil {
 			return err
 		}
 	}
  `
+
 	tpl := template.New(fmt.Sprintf("%s", time.Now().String()))
 	tpl.Parse(uninstall)
 	buf := &bytes.Buffer{}

--- a/codify/poddisruptionbudget.go
+++ b/codify/poddisruptionbudget.go
@@ -29,37 +29,38 @@ import (
 	"text/template"
 	"time"
 
+	policyv1 "k8s.io/api/policy/v1beta1"
+
 	"github.com/kris-nova/logger"
-	networkingv1 "k8s.io/api/networking/v1"
 )
 
-type Ingress struct {
-	KubeObject *networkingv1.Ingress
+type PodDisruptionBudget struct {
+	KubeObject *policyv1.PodDisruptionBudget
 	GoName     string
 }
 
-func NewIngress(obj *networkingv1.Ingress) *Ingress {
+func NewPodDisruptionBudget(obj *policyv1.PodDisruptionBudget) *PodDisruptionBudget {
 	obj.ObjectMeta = cleanObjectMeta(obj.ObjectMeta)
-	return &Ingress{
+	obj.Status = policyv1.PodDisruptionBudgetStatus{}
+	return &PodDisruptionBudget{
 		KubeObject: obj,
 		GoName:     goName(obj.Name),
 	}
 }
 
-func (k Ingress) Install() (string, []string) {
+func (k PodDisruptionBudget) Install() (string, []string) {
 	l, packages := Literal(k.KubeObject)
 	install := fmt.Sprintf(`
-	{{ .GoName }}Ingress := %s
-	x.objects = append(x.objects, {{ .GoName }}Ingress)
+	{{ .GoName }}PodDisruptionBudget := %s
+	x.objects = append(x.objects, {{ .GoName }}PodDisruptionBudget)
 
 	if client != nil {
-		_, err = client.NetworkingV1().Ingresses("{{ .KubeObject.Namespace }}").Create(context.TODO(), {{ .GoName }}Ingress, v1.CreateOptions{})
+		_, err = client.PolicyV1beta1().PodDisruptionBudgets("{{ .KubeObject.Namespace }}").Create(context.TODO(), {{ .GoName }}PodDisruptionBudget, v1.CreateOptions{})
 		if err != nil {
 			return err
 		}
 	}
 `, l)
-
 	tpl := template.New(fmt.Sprintf("%s", time.Now().String()))
 	tpl.Parse(install)
 	buf := &bytes.Buffer{}
@@ -68,13 +69,13 @@ func (k Ingress) Install() (string, []string) {
 	if err != nil {
 		logger.Debug(err.Error())
 	}
-	return alias(buf.String(), "networkingv1"), packages
+	return alias(buf.String(), "policyv1"), packages
 }
 
-func (k Ingress) Uninstall() string {
+func (k PodDisruptionBudget) Uninstall() string {
 	uninstall := `
 	if client != nil {
-		err = client.NetworkingV1().Ingress("{{ .KubeObject.Namespace }}").Delete(context.TODO(), "{{ .KubeObject.Name }}", metav1.DeleteOptions{})
+		err = client.PolicyV1beta1().PodDisruptionBudgets("{{ .KubeObject.Namespace }}").Delete(context.TODO(), "{{ .KubeObject.Name }}", metav1.DeleteOptions{})
 		if err != nil {
 			return err
 		}

--- a/codify/statefulset.go
+++ b/codify/statefulset.go
@@ -65,7 +65,7 @@ func (k StatefulSet) Install() (string, []string) {
 	x.objects = append(x.objects, {{ .GoName }}StatefulSet)
 
 	if client != nil {
-		_, err = client.AppsV1().StatefulSets("{{ .KubeObject.Namespace }}").Create(context.TODO(), {{ .GoName }}Deployment, v1.CreateOptions{})
+		_, err = client.AppsV1().StatefulSets("{{ .KubeObject.Namespace }}").Create(context.TODO(), {{ .GoName }}StatefulSet, v1.CreateOptions{})
 		if err != nil {
 			return err
 		}

--- a/compile.go
+++ b/compile.go
@@ -29,6 +29,7 @@ import (
 	"io/ioutil"
 	"os"
 	"os/exec"
+	"sync"
 )
 
 type Source struct {
@@ -40,9 +41,13 @@ type Program struct {
 	File   *os.File
 }
 
+var mtx sync.Mutex
+
 // Execute will execute a compiled NAML program
 // stdout, stderr, err := program.Execute([]string{""})
 func (p *Program) Execute(flags []string) (*bytes.Buffer, *bytes.Buffer, error) {
+	mtx.Lock()
+	defer mtx.Unlock()
 	cmd := exec.Command(p.File.Name(), flags...)
 	stdout := &bytes.Buffer{}
 	stderr := &bytes.Buffer{}

--- a/embed_gen.go
+++ b/embed_gen.go
@@ -184,4 +184,3 @@ func (x *{{ .AppNameTitle }}) Objects() []runtime.Object {
 	return x.objects
 }
 `
-

--- a/tests/manifest_test.go
+++ b/tests/manifest_test.go
@@ -27,7 +27,6 @@ import (
 	"fmt"
 	"io/ioutil"
 	"path/filepath"
-	"sync"
 	"testing"
 
 	"github.com/kris-nova/naml"
@@ -38,20 +37,14 @@ func TestManifests(t *testing.T) {
 	if err != nil {
 		t.Errorf("unable to list test_nivenly.yaml directory: %v", err)
 	}
-	wg := sync.WaitGroup{}
 	for _, file := range files {
-		wg.Add(1)
-		go func(name string) {
-			defer wg.Done()
-			t.Logf("testing [%s]", name)
-			err := generateCompileRunYAML(filepath.Join("manifests", file.Name()))
-			if err != nil {
-				t.Errorf(err.Error())
-				t.FailNow()
-			}
-		}(file.Name())
+		t.Logf("testing [%s]", file.Name())
+		err := generateCompileRunYAML(filepath.Join("manifests", file.Name()))
+		if err != nil {
+			t.Errorf(err.Error())
+			t.FailNow()
+		}
 	}
-	wg.Wait()
 	t.Logf("Manifest tests complete")
 }
 
@@ -86,7 +79,7 @@ func generateCompileRunYAML(filename string) error {
 	if err != nil {
 		return fmt.Errorf("unable to compile: %s: %v", filename, err)
 	}
-	_, stderr, err := program.Execute([]string{""})
+	_, stderr, err := program.Execute([]string{"list"})
 	if stderr.Len() > 0 {
 		return fmt.Errorf("failed executing binary: %s: %v: %s", filename, err, stderr.String())
 	}

--- a/tests/manifest_test.go
+++ b/tests/manifest_test.go
@@ -79,7 +79,8 @@ func generateCompileRunYAML(filename string) error {
 	if err != nil {
 		return fmt.Errorf("unable to compile: %s: %v", filename, err)
 	}
-	_, stderr, err := program.Execute([]string{"list"})
+	//	_, stderr, err := program.Execute([]string{"list"})
+	_, stderr, err := program.Execute([]string{""})
 	if stderr.Len() > 0 {
 		return fmt.Errorf("failed executing binary: %s: %v: %s", filename, err, stderr.String())
 	}

--- a/tests/manifests/test_ingress_service.yaml
+++ b/tests/manifests/test_ingress_service.yaml
@@ -1,0 +1,20 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: nginx-thing
+  annotations:
+   nginx.ingress.kubernetes.io/rewrite-target: /$2
+  labels:
+    app: nginx-thing
+spec:
+  ingressClassName: nginx
+  rules:
+    - http:
+       paths:
+         - pathType: Prefix
+           path: /nginx-thing(/|$)(.*)
+           backend:
+             service:
+               name: nginx-thing
+               port:
+                 number: 8080

--- a/tests/manifests/test_jaeger_helm.yaml
+++ b/tests/manifests/test_jaeger_helm.yaml
@@ -3,11 +3,11 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: RELEASE-NAME-jaeger-agent
+  name: naml-jaeger-agent
   labels:
     helm.sh/chart: jaeger-0.51.4
     app.kubernetes.io/name: jaeger
-    app.kubernetes.io/instance: RELEASE-NAME
+    app.kubernetes.io/instance: naml
     app.kubernetes.io/version: "1.28.0"
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/component: agent
@@ -17,11 +17,11 @@ automountServiceAccountToken: false
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: RELEASE-NAME-jaeger-cassandra-schema
+  name: naml-jaeger-cassandra-schema
   labels:
     helm.sh/chart: jaeger-0.51.4
     app.kubernetes.io/name: jaeger
-    app.kubernetes.io/instance: RELEASE-NAME
+    app.kubernetes.io/instance: naml
     app.kubernetes.io/version: "1.28.0"
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/component: cassandra-schema
@@ -31,11 +31,11 @@ automountServiceAccountToken: true
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: RELEASE-NAME-jaeger-collector
+  name: naml-jaeger-collector
   labels:
     helm.sh/chart: jaeger-0.51.4
     app.kubernetes.io/name: jaeger
-    app.kubernetes.io/instance: RELEASE-NAME
+    app.kubernetes.io/instance: naml
     app.kubernetes.io/version: "1.28.0"
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/component: collector
@@ -45,11 +45,11 @@ automountServiceAccountToken: false
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: RELEASE-NAME-jaeger-query
+  name: naml-jaeger-query
   labels:
     helm.sh/chart: jaeger-0.51.4
     app.kubernetes.io/name: jaeger
-    app.kubernetes.io/instance: RELEASE-NAME
+    app.kubernetes.io/instance: naml
     app.kubernetes.io/version: "1.28.0"
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/component: query
@@ -59,11 +59,11 @@ automountServiceAccountToken: false
 apiVersion: v1
 kind: Secret
 metadata:
-  name: RELEASE-NAME-jaeger-cassandra
+  name: naml-jaeger-cassandra
   labels:
     helm.sh/chart: jaeger-0.51.4
     app.kubernetes.io/name: jaeger
-    app.kubernetes.io/instance: RELEASE-NAME
+    app.kubernetes.io/instance: naml
     app.kubernetes.io/version: "1.28.0"
     app.kubernetes.io/managed-by: Helm
 type: Opaque
@@ -74,11 +74,11 @@ data:
 apiVersion: v1
 kind: Service
 metadata:
-  name: RELEASE-NAME-cassandra
+  name: naml-cassandra
   labels:
     app: cassandra
     chart: cassandra-0.15.3
-    release: RELEASE-NAME
+    release: naml
     heritage: Helm
 spec:
   clusterIP: None
@@ -101,17 +101,17 @@ spec:
     targetPort: 9160
   selector:
     app: cassandra
-    release: RELEASE-NAME
+    release: naml
 ---
 # Source: jaeger/templates/agent-svc.yaml
 apiVersion: v1
 kind: Service
 metadata:
-  name: RELEASE-NAME-jaeger-agent
+  name: naml-jaeger-agent
   labels:
     helm.sh/chart: jaeger-0.51.4
     app.kubernetes.io/name: jaeger
-    app.kubernetes.io/instance: RELEASE-NAME
+    app.kubernetes.io/instance: naml
     app.kubernetes.io/version: "1.28.0"
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/component: agent
@@ -140,18 +140,18 @@ spec:
   type: ClusterIP
   selector:
     app.kubernetes.io/name: jaeger
-    app.kubernetes.io/instance: RELEASE-NAME
+    app.kubernetes.io/instance: naml
     app.kubernetes.io/component: agent
 ---
 # Source: jaeger/templates/collector-svc.yaml
 apiVersion: v1
 kind: Service
 metadata:
-  name: RELEASE-NAME-jaeger-collector
+  name: naml-jaeger-collector
   labels:
     helm.sh/chart: jaeger-0.51.4
     app.kubernetes.io/name: jaeger
-    app.kubernetes.io/instance: RELEASE-NAME
+    app.kubernetes.io/instance: naml
     app.kubernetes.io/version: "1.28.0"
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/component: collector
@@ -170,7 +170,7 @@ spec:
     targetPort: admin
   selector:
     app.kubernetes.io/name: jaeger
-    app.kubernetes.io/instance: RELEASE-NAME
+    app.kubernetes.io/instance: naml
     app.kubernetes.io/component: collector
   type: ClusterIP
 ---
@@ -178,11 +178,11 @@ spec:
 apiVersion: v1
 kind: Service
 metadata:
-  name: RELEASE-NAME-jaeger-query
+  name: naml-jaeger-query
   labels:
     helm.sh/chart: jaeger-0.51.4
     app.kubernetes.io/name: jaeger
-    app.kubernetes.io/instance: RELEASE-NAME
+    app.kubernetes.io/instance: naml
     app.kubernetes.io/version: "1.28.0"
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/component: query
@@ -198,7 +198,7 @@ spec:
     targetPort: admin
   selector:
     app.kubernetes.io/name: jaeger
-    app.kubernetes.io/instance: RELEASE-NAME
+    app.kubernetes.io/instance: naml
     app.kubernetes.io/component: query
   type: ClusterIP
 ---
@@ -206,11 +206,11 @@ spec:
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
-  name: RELEASE-NAME-jaeger-agent
+  name: naml-jaeger-agent
   labels:
     helm.sh/chart: jaeger-0.51.4
     app.kubernetes.io/name: jaeger
-    app.kubernetes.io/instance: RELEASE-NAME
+    app.kubernetes.io/instance: naml
     app.kubernetes.io/version: "1.28.0"
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/component: agent
@@ -218,21 +218,21 @@ spec:
   selector:
     matchLabels:
       app.kubernetes.io/name: jaeger
-      app.kubernetes.io/instance: RELEASE-NAME
+      app.kubernetes.io/instance: naml
       app.kubernetes.io/component: agent
   template:
     metadata:
       labels:
         app.kubernetes.io/name: jaeger
-        app.kubernetes.io/instance: RELEASE-NAME
+        app.kubernetes.io/instance: naml
         app.kubernetes.io/component: agent
     spec:
       securityContext:
         {}
       dnsPolicy: ClusterFirst
-      serviceAccountName: RELEASE-NAME-jaeger-agent
+      serviceAccountName: naml-jaeger-agent
       containers:
-      - name: RELEASE-NAME-jaeger-agent
+      - name: naml-jaeger-agent
         securityContext:
           {}
         image: jaegertracing/jaeger-agent:1.28.0
@@ -240,7 +240,7 @@ spec:
         args:
         env:
           - name: REPORTER_GRPC_HOST_PORT
-            value: RELEASE-NAME-jaeger-collector:14250
+            value: naml-jaeger-collector:14250
         ports:
         - name: zipkin-compact
           containerPort: 5775
@@ -274,11 +274,11 @@ spec:
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: RELEASE-NAME-jaeger-collector
+  name: naml-jaeger-collector
   labels:
     helm.sh/chart: jaeger-0.51.4
     app.kubernetes.io/name: jaeger
-    app.kubernetes.io/instance: RELEASE-NAME
+    app.kubernetes.io/instance: naml
     app.kubernetes.io/version: "1.28.0"
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/component: collector
@@ -287,7 +287,7 @@ spec:
   selector:
     matchLabels:
       app.kubernetes.io/name: jaeger
-      app.kubernetes.io/instance: RELEASE-NAME
+      app.kubernetes.io/instance: naml
       app.kubernetes.io/component: collector
   template:
     metadata:
@@ -295,14 +295,14 @@ spec:
         checksum/config-env: 75a11da44c802486bc6f65640aa48a730f0f684c5c07a42ba3cd1735eb3fb070
       labels:
         app.kubernetes.io/name: jaeger
-        app.kubernetes.io/instance: RELEASE-NAME
+        app.kubernetes.io/instance: naml
         app.kubernetes.io/component: collector
     spec:
       securityContext:
         {}
-      serviceAccountName: RELEASE-NAME-jaeger-collector
+      serviceAccountName: naml-jaeger-collector
       containers:
-      - name: RELEASE-NAME-jaeger-collector
+      - name: naml-jaeger-collector
         securityContext:
           {}
         image: jaegertracing/jaeger-collector:1.28.0
@@ -313,7 +313,7 @@ spec:
           - name: SPAN_STORAGE_TYPE
             value: cassandra
           - name: CASSANDRA_SERVERS
-            value: RELEASE-NAME-cassandra
+            value: naml-cassandra
           - name: CASSANDRA_PORT
             value: "9042"
           
@@ -324,7 +324,7 @@ spec:
           - name: CASSANDRA_PASSWORD
             valueFrom:
               secretKeyRef:
-                name: RELEASE-NAME-jaeger-cassandra
+                name: naml-jaeger-cassandra
                 key: password
         ports:
         - containerPort: 14250
@@ -355,11 +355,11 @@ spec:
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: RELEASE-NAME-jaeger-query
+  name: naml-jaeger-query
   labels:
     helm.sh/chart: jaeger-0.51.4
     app.kubernetes.io/name: jaeger
-    app.kubernetes.io/instance: RELEASE-NAME
+    app.kubernetes.io/instance: naml
     app.kubernetes.io/version: "1.28.0"
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/component: query
@@ -368,20 +368,20 @@ spec:
   selector:
     matchLabels:
       app.kubernetes.io/name: jaeger
-      app.kubernetes.io/instance: RELEASE-NAME
+      app.kubernetes.io/instance: naml
       app.kubernetes.io/component: query
   template:
     metadata:
       labels:
         app.kubernetes.io/name: jaeger
-        app.kubernetes.io/instance: RELEASE-NAME
+        app.kubernetes.io/instance: naml
         app.kubernetes.io/component: query
     spec:
       securityContext:
         {}
-      serviceAccountName: RELEASE-NAME-jaeger-query
+      serviceAccountName: naml-jaeger-query
       containers:
-      - name: RELEASE-NAME-jaeger-query
+      - name: naml-jaeger-query
         securityContext:
           {}
         image: jaegertracing/jaeger-query:1.28.0
@@ -392,7 +392,7 @@ spec:
           - name: SPAN_STORAGE_TYPE
             value: cassandra
           - name: CASSANDRA_SERVERS
-            value: RELEASE-NAME-cassandra
+            value: naml-cassandra
           - name: CASSANDRA_PORT
             value: "9042"
           
@@ -403,7 +403,7 @@ spec:
           - name: CASSANDRA_PASSWORD
             valueFrom:
               secretKeyRef:
-                name: RELEASE-NAME-jaeger-cassandra
+                name: naml-jaeger-cassandra
                 key: password
           - name: JAEGER_AGENT_PORT
             value: "6831"
@@ -425,7 +425,7 @@ spec:
           httpGet:
             path: /
             port: admin
-      - name: RELEASE-NAME-jaeger-agent-sidecar
+      - name: naml-jaeger-agent-sidecar
         securityContext:
           {}
         image: jaegertracing/jaeger-agent:1.28.0
@@ -433,7 +433,7 @@ spec:
         args:
         env:
         - name: REPORTER_GRPC_HOST_PORT
-          value: RELEASE-NAME-jaeger-collector:14250
+          value: naml-jaeger-collector:14250
         ports:
         - name: admin
           containerPort: 14271
@@ -457,18 +457,18 @@ spec:
 apiVersion: apps/v1
 kind: StatefulSet
 metadata:
-  name: RELEASE-NAME-cassandra
+  name: naml-cassandra
   labels:
     app: cassandra
     chart: cassandra-0.15.3
-    release: RELEASE-NAME
+    release: naml
     heritage: Helm
 spec:
   selector:
     matchLabels:
       app: cassandra
-      release: RELEASE-NAME
-  serviceName: RELEASE-NAME-cassandra
+      release: naml
+  serviceName: naml-cassandra
   replicas: 3
   podManagementPolicy: OrderedReady
   updateStrategy:
@@ -477,18 +477,18 @@ spec:
     metadata:
       labels:
         app: cassandra
-        release: RELEASE-NAME
+        release: naml
     spec:
       hostNetwork: false
       containers:
-      - name: RELEASE-NAME-cassandra
+      - name: naml-cassandra
         image: "cassandra:3.11.6"
         imagePullPolicy: "IfNotPresent"
         resources:
           {}
         env:
         - name: CASSANDRA_SEEDS
-          value: "RELEASE-NAME-cassandra-0.RELEASE-NAME-cassandra.default.svc.cluster.local"
+          value: "naml-cassandra-0.naml-cassandra.default.svc.cluster.local"
         - name: MAX_HEAP_SIZE
           value: "2048M"
         - name: HEAP_NEWSIZE
@@ -550,11 +550,11 @@ spec:
 apiVersion: batch/v1
 kind: Job
 metadata:
-  name: RELEASE-NAME-jaeger-cassandra-schema
+  name: naml-jaeger-cassandra-schema
   labels:
     helm.sh/chart: jaeger-0.51.4
     app.kubernetes.io/name: jaeger
-    app.kubernetes.io/instance: RELEASE-NAME
+    app.kubernetes.io/instance: naml
     app.kubernetes.io/version: "1.28.0"
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/component: cassandra-schema
@@ -562,13 +562,13 @@ spec:
   activeDeadlineSeconds: 300
   template:
     metadata:
-      name: RELEASE-NAME-jaeger-cassandra-schema
+      name: naml-jaeger-cassandra-schema
     spec:
       securityContext:
         {}
-      serviceAccountName: RELEASE-NAME-jaeger-cassandra-schema
+      serviceAccountName: naml-jaeger-cassandra-schema
       containers:
-      - name: RELEASE-NAME-jaeger-cassandra-schema
+      - name: naml-jaeger-cassandra-schema
         image: jaegertracing/jaeger-cassandra-schema:1.28.0
         imagePullPolicy: IfNotPresent
         securityContext:
@@ -576,7 +576,7 @@ spec:
         env:
           
           - name: CASSANDRA_SERVERS
-            value: RELEASE-NAME-cassandra
+            value: naml-cassandra
           - name: CASSANDRA_PORT
             value: "9042"
           
@@ -587,10 +587,10 @@ spec:
           - name: CASSANDRA_PASSWORD
             valueFrom:
               secretKeyRef:
-                name: RELEASE-NAME-jaeger-cassandra
+                name: naml-jaeger-cassandra
                 key: password
           - name: CQLSH_HOST
-            value: RELEASE-NAME-cassandra
+            value: naml-cassandra
           
           - name: DATACENTER
             value: "dc1"

